pkg/relay/ethereum/chain.go

```diff
@@ -94,6 +94,15 @@ func (c *Chain) LatestHeight() (ibcexported.Height, error) {
 	return clienttypes.NewHeight(0, bn), nil
 }
 
+func (c *Chain) Timestamp(height ibcexported.Height) (time.Time, error) {
+	ht := big.NewInt(int64(height.GetRevisionHeight()))
+	if header, err := c.client.HeaderByNumber(context.TODO(), ht); err != nil {
+		return time.Time{}, err
+	} else {
+		return time.Unix(int64(header.Time), 0), err
```

Comment:
nits: As elsewhere, I prefer to return `nil` explicitly instead of `err`

Reply:
Me too. In fact, this is a mistake. I will fix it immediately. Thanks.

Reply:
Fixed.